python/python/res/simulator/batch_simulator.py

```diff
@@ -87,7 +87,7 @@ def _setup_case(self, case, file_system):
                 node.save(file_system, node_id)
 
 
-    def start(self, case_name, case_data):
+    def start(self, case_name, case_data, start_callback=None):
```

Comment:
This looks very good, but is there a reason you named if `start_callback` instead of `callback`?   Are we going to support more callbacks to this function?